Source/Charts/Utils/ChartUtils.swift

```diff
@@ -287,60 +287,4 @@ open class ChartUtils
     {
         return _defaultValueFormatter
     }
-
-    /// MARK: - Bridging functions
-    
-    internal class func bridgedObjCGetNSUIColorArray (swift array: [NSUIColor?]) -> [NSObject]
```

Comment:
why this is included?

Reply:
These only existed to support the deprecated functions. Since I am removing the deprecated functions in this PR, these methods are no longer needed.

Reply:
Also, this could have been written as `let myObjArray = myColorArray as! [NSObject]`